core/templates/pages/contributor-dashboard-page/services/contribution-and-review-backend-api.service.spec.ts

```diff
@@ -102,36 +102,38 @@ describe('Contribution and review backend API service', () => {
 
     it('should fetch reviewable question suggestions', fakeAsync(() => {
       spyOn(carbas, 'fetchReviewableSuggestionsAsync').and.callThrough();
-      const url = '/getreviewablesuggestions/skill/add_question';
+      const url = '/getreviewablesuggestions' +
+        '/skill/add_question?topic_name=all';
 
       carbas.fetchSuggestionsAsync(
-        'REVIEWABLE_QUESTION_SUGGESTIONS'
-      ).then(successHandler, failureHandler);
+        'REVIEWABLE_QUESTION_SUGGESTIONS',
+        'all').then(successHandler, failureHandler);
       const req = http.expectOne(url);
       expect(req.request.method).toEqual('GET');
       req.flush(suggestionsBackendObject);
       flushMicrotasks();
 
       expect(carbas.fetchReviewableSuggestionsAsync)
-        .toHaveBeenCalledWith('skill', 'add_question');
+        .toHaveBeenCalledWith('skill', 'add_question', 'all');
```

Comment:
Is it possible to create some translation suggestions that are reviewable and some that are not and then write unit tests instead of just using `'all'`?

Reply:
I think all translation suggestions are reviewable, was what you wanted suggestions linked to different topics? Suggestions are linked to explorations which are linked stories which are linked to topics. You can see the backend logic here: https://github.com/oppia/oppia/pull/15070/files

Reply:
Ah, sorry. I wanted suggestions linked to different topics.